build/dependencies.props

```diff
@@ -5,26 +5,34 @@
   </PropertyGroup>
   <PropertyGroup Label="Package Versions">
     <!-- EFCore.MySql Dependencies -->
-    <MicrosoftEntityFrameworkCoreRelationalVersion>2.2.0</MicrosoftEntityFrameworkCoreRelationalVersion>
-    <MySqlConnectorVersion>0.51.1</MySqlConnectorVersion>
+    <MicrosoftEntityFrameworkCoreRelationalVersion>3.0.0-rc1.19452.2</MicrosoftEntityFrameworkCoreRelationalVersion>
+    <MySqlConnectorVersion>0.57.0-rc1</MySqlConnectorVersion>
```

Comment:
Thanks! I plan to have a stable version out very soon; just waiting for the final SDK.

Reply:
Looking forward to it.
I might open an issue about the `NO_BACKSLASH_ESCAPES` support in conjunction with parameters for MySqlConnect tomorrow. Just have to do a bit more research on that before.